tests/functional/healthchecks/master.json

```diff
@@ -0,0 +1,6 @@
+{
+    "tests": {
+        "files": [ "checkRoutes.js" ],
```

Comment:
I assume we will have more files than this so probably makes sense to add a test directory, put checkRoutes.js in it and reference the test directory here.